src/vs/workbench/parts/extensions/electron-browser/media/extensionsViewlet.css

```diff
@@ -69,6 +69,10 @@
 	box-sizing: border-box;
 }
 
+.extensions-viewlet > .extensions .monaco-list-row.disabled > .bookmark {
```

Comment:
let's merge this with the ruleset in line 200 so that all the "disabled" rules are in 1 place.